### DISCUSSION TITLE
Changing top level "Thanks!" page / Redirect

### DISCUSSION THREAD

**Discussion by penguinswin3** [cat-box/aws-foundry-ssl#30] [Q&A] [answered]
I have recently deployed foundry successfully and everything has been working well! I am looking to change how the base welcome page works however. 

When navigating to \<myDomain\>.org, I am sent to a nice looking Thanks page. When I navigate to foundry.\<myDomain\>.org, I end up at the foundry login page. Cool! 

I am looking to set up a redirect on the <myDomain>.org page to go to either a different subdomain (www.\<myDomain\>.org or foundry.\<myDomain\>.org).

I am new to AWS and am kinda lost on how to do this, mostly because I do not actually know where the Thanks page is hosted... 

Thank you in advance for any possible guidance!

**Comment by cat-box** [comment, marked as answer]
SSH into the ec2-instance and navigate to `/usr/share/nginx/html`. You'll have to replace our webpage with your own html files :)

If you need instructions on how to SSH, the wiki has a step by step guide!
